include/grpc/grpc_security.h

```diff
@@ -323,6 +323,10 @@ GRPCAPI grpc_channel *grpc_secure_channel_create(
     grpc_channel_credentials *creds, const char *target,
     const grpc_channel_args *args, void *reserved);
 
+GRPCAPI grpc_channel *grpc_custom_secure_channel_create(
```

Comment:
rename to: grpc_cronet_secure_channel_create
move to: include/grpc/grpc_cronet.h